odoo/tests/common.py

```diff
@@ -1153,6 +1153,16 @@ def navigate_to(self, url, wait_stop=False):
             self._websocket_wait_event('Page.frameStoppedLoading', params={'frameId': frame_id})
 
     def clear(self):
+        clear_service_workers = """
+        if ('serviceWorker' in navigator) {
+            navigator.serviceWorker.getRegistrations().then(
+                (registrations) => {registrations.map(r => r.unregister())}
+            )
+        }
+        """
+        cl_id = self._websocket_send('Runtime.evaluate', params={'expression': clear_service_workers})
+        self._websocket_wait_id(cl_id)
+
```

Comment:
Nice, but did you try this DevTools Protocol command ?
`self._websocket_send('ServiceWorker.stopAllWorkers ')`

This avoid to execute js.
See [Documentation](https://chromedevtools.github.io/devtools-protocol/tot/ServiceWorker/)

Reply:
@d-fence Hi :)
Good catch for stopAllWorkers, but it doesn't seems to work on the runbot
See https://runbot.odoo.com/runbot/build/5162353

And here's a standard build, all the test passed :) 
https://runbot.odoo.com/runbot/build/5162474

Reply:
Probably need a [`ServiceWorker.enable`](https://chromedevtools.github.io/devtools-protocol/tot/ServiceWorker/#method-enable) to say that we will use it.

See https://github.com/odoo/odoo/blob/9dc9e00879d018cc9b964eccbda1b6caca480e7e/odoo/tests/common.py#L704-L707

Reply:
@d-fence Do I still miss something ? https://runbot.odoo.com/runbot/build/5163476
(work in most cases, but still failed sometimes :/ Like is it didn't always stop the service worker properly)

Reply:
Ok, we should stick with your first fix as it was working as expected.
The feature is marked as `experimental` ... as most of their features.
This protocol is a moving target, I could try to investigate that in another PR.